src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -264,27 +258,53 @@ private int getWindowWidth() {
         return appiumDriver.manage().window().getSize().width;
     }
 
-    private void swipe(int height, int fromWidth, int toWidth) {
-        AppiumDriver appiumDriver = (AppiumDriver) this.driver;
-        throw new NotImplementedException("To be migrated to appium 2.0");
-// todo - to be implemented in appium 2.0
-//        TouchAction touchAction = new TouchAction(appiumDriver);
-//        touchAction.press(PointOption.point(new Point(fromWidth, height)))
-//                   .waitAction(WaitOptions.waitOptions(Duration.ofSeconds(1)))
-//                   .moveTo(PointOption.point(new Point(toWidth, height)))
-//                   .release()
-//                   .perform();
+    private boolean checkPercentagesAreValid(int... percentages) {
+        return Arrays.stream(percentages).allMatch(percentage -> percentage >= 0 && percentage <= 100);
+    }
+
+    public void swipeRight() {
+        int height = getWindowHeight() / 2;
+        int fromWidth = (int) (getWindowWidth() * 0.2);
+        int toWidth = (int) (getWindowWidth() * 0.7);
```

Comment:
float value for swiping right calculations are 0.2 and 0.7
float value for swiping left calculations are 0.3 and 0.8
use either both as 0.2, 0.7 or 0.3, 0.8 (whatever is the basis of the calculation)
why are they different, consider making then equal

Reply:
These cant be equal.

Anand, Ankit suggested swiping from the edge of the screen. we are keeping a buffer of 2 points on both edges. thus, the current calculation is being pre decided